src/docs.md

```diff
@@ -335,6 +335,9 @@ Campaign uid to filter results by campaign.
 `campaignUids` (array)<br>
 Campaign uids to filter results by multiple campaigns.
 
+`groupValues` (array)<br>
```

Comment:
We should look into the markdown parser config. I'm pretty sure you can configure it to convert `\n` to `<br />`.